### ISSUE TITLE
Cannot use `esc` key to close Branch list

### ISSUE THREAD

**Issue by tierninho** [desktop/desktop#7177]
Using the `esc` key to close the `branches-container` is still possible, however I found an edge case where if you click into the white space in the Pull Request list, then using `esc` won't close the list.

<img width="351" alt="Screen Shot 2019-03-28 at 3 03 08 PM" src="https://user-images.githubusercontent.com/14828183/55202233-acdc1b00-516a-11e9-91c2-a161b73c8515.png">

v1.7.0-beta1, mac

**Comment by b0810** [comment]
Bonjour ++prenom++,, 

Vous cherchez un nouveau site pouvant vous aider à gagner des millions au bout de 3 mois de travail acharné? 

Vous tombez bien parce que j"ai un tout nouveau site qui peut vous générer des millions de dollars en trois mois.
Cliquez sur ce lien si vous le voulez bien Builderall Leverage Affiliate System | Builderall Leveraged Affiliate System

| 
| 
|  | 
Builderall Leverage Affiliate System | Builderall Leveraged Affiliate Sy...

Builderall Business's Leveraged Affiliate System gives you a leveraged gain and solid recurring income based on ...
 |

 |

 |



Cordialement, 

Borath YUN




Envoyé depuis Yahoo Mail. Obtenir l’application 

    Le vendredi 29 mars 2019 à 02:04:50 UTC+1, Tiernan L <notifications@github.com> a écrit :  
 
 
Using the esc key to close the branches-container, namely the Pull Request list is still possible, however I found an edge case where if you click into the white space, then using esc to close the list doesn't work.



v1.7.0-beta1, mac
